diff -urpN pyload.orig/module/setup.py pyload.new/module/setup.py
--- pyload.orig/module/setup.py	2011-10-04 14:55:18.000000000 +0200
+++ pyload.new/module/setup.py	2011-11-24 04:38:05.053312713 +0100
@@ -238,7 +238,7 @@ class Setup():
 
         try:
             import jinja2
-            if jinja2.__version__ and "unknown" not in jinja2.__version__ and not jinja2.__version__.startswith("2.5"): #@TODO: could be to new aswell
+            if jinja2.__version__ and "unknown" not in jinja2.__version__ and not (jinja2.__version__.startswith("2.5") or jinja2.__version__.startswith("2.6") ): #@TODO: could be to new aswell
                 print _("Your installed jinja2 version %s seems too old.") % jinja2.__version__
                 print _("You can safely continue but if the webinterface is not working,")
                 print _("please upgrade or deinstall it, pyLoad includes a sufficient jinja2 libary.")
